skinning/skinning_test_simd.cpp

```diff
@@ -162,7 +162,7 @@ void calculateVerticesAndNormals_SSE_intrinsics(
       _mm_storeh_pi((__m64*)output_vertex, sum2);
 
       __m128 sum3 = _mm_shuffle_ps(sum2, sum2, _MM_SHUFFLE(1, 1, 1, 1));
-      sum3 = _mm_add_ss(sum2, sum3);
+      sum3 = _mm_add_ps(sum2, sum3);
```

Comment:
why did you change this? _mm_add_ss on several CPUs is faster, so this isn't quite a fair comparison.